File: 1791. Find Center of Star Graph.java
class Solution {
	public int findCenter(int[][] edges) {
        int a=edges[0][0];
        int b=edges[0][1];
        int flag=0;
        for(int i=1;i<edges.length;i++)
        {
            if(edges[i][0]!=a && edges[i][1]!=a)
            {
                flag=1;
                break;
            }
        }
        if(flag==1)
            return b;
        else
            return a;
	}
}

File: 1791. Find Center of Star Graph.kt
class Solution {
    fun findCenter(edges: Array<IntArray>): Int {
        val map = sortedMapOf<Int, Int>()
        for(edge in edges) {
            map[edge[0]] = map.getOrDefault(edge[0], 0) + 1
            map[edge[1]] = map.getOrDefault(edge[1], 0) + 1
        }
        for(key in map.keys) {
            if(map[key]!! == edges.size) {
                return key
            }
        }
        return -1
    }
}
